### DISCUSSION TITLE
[AppStartup] MAUI app Startup is slower than Xamarin

### DISCUSSION THREAD

**Discussion by emako** [dotnet/maui#15693] [General]
### Description

The startup speed of MAUI's app is much slower than that of Xamarin.

For the demo project of "Hello World", Xamarin starts an application in 3 seconds, and MAUI takes at least 8 seconds, which is unbearable. It tested in iOS.

I hope to provide a "Hello World" project demo for MAUI with the same startup time level as Xamarin.

### Public API Changes

Nothings

### Intended Use-Case

Nothings

**Comment by AlleSchonWeg** [comment]
Perhaps the same: https://github.com/dotnet/maui/issues/11392

**Comment by mohachouch** [comment]
Debug or Release mode?

**Comment by emako** [comment]
Release mode and build into iOS
